samples/javascript_nodejs/11.qnamaker/README.md

```diff
@@ -1,53 +1,59 @@
 # QnA Maker sample
+
 Bot Framework v4 QnA Maker bot sample
 
 This bot has been created using [Microsoft Bot Framework][1], it shows how to create a bot that uses the [QnA Maker Cognitive AI][19] service.
 
 The [QnA Maker Service][19] enables you to build, train and publish a simple question and answer bot based on FAQ URLs, structured documents or editorial content in minutes. In this sample, we demonstrate how to use the QnA Maker service to answer questions based on a FAQ text file used as input.
 
 ## Prerequisites
+
 This samples requires prerequisites in order to run.
+
 - [Required Prerequisites][41]
 
+### Create a QnAMaker Application to enable QnA Knowledge Bases
+
+QnA knowledge base setup and application configuration steps can be found [here][7].
+
 # To try this sample
+
 - Clone the repository
+
     ```bash
     git clone https://github.com/microsoft/botbuilder-samples.git
     ```
-- In a terminal, navigate to `samples/javascript_nodejs/11.qnamaker`
+
+- In a terminal, navigate to `samples/javascript_nodejs/13.core-bot`
+
     ```bash
     cd samples/javascript_nodejs/11.qnamaker
     ```
+
 - Install modules
+
     ```bash
     npm install
     ```
-- Setup QnA Maker Knowledgebase
 
-    Assuming prerequisites have been installed:
-    ```bash
-    # log into Azure
-    az login
-    ```
-    ```bash
-    # set you Azure subscription
-    az account set --subscription "<azure-subscription>"
-    ```
-    ```bash
-    # Create QnA Maker service application
-    msbot clone services --name "<your_bot_name>" --code-dir "." --location westus --sdkLanguage "Node" --folder deploymentScripts/msbotClone --verbose
-    ```
-- Start the bot
+- Setup QnAMaker
+
+    The prerequisite outlined above contain the steps necessary to provision a QnA Knowledge Base on www.qnamaker.ai.  Refer to _Create a QnAMaker Application to enable QnA Knowledge Bases_ above for directions to setup and configure LUIS.
```

Comment:
If you're going to reference a separate prereqs file, you should link to it here.

Reply:
Done.